pkgs/top-level/aliases.nix

```diff
@@ -540,6 +540,8 @@ mapAliases ({
   surf-webkit2 = surf; # added 2017-04-02
   sup = throw "deprecated in 2019-09-10: abandoned by upstream";
   system_config_printer = system-config-printer;  # added 2016-01-03
+  systemd-cryptsetup-generator = throw "included into a generic systemd already"; # added 2020-04-30
+  systemd_with_lvm2 = throw "please add lvm2 system generators by setting systemd.packages = [ pkgs.lvm2 ];"; # added 2019-04-30
```

Comment:
This should communicate the attribute name where the throw is coming from.

Reply:
done.